### ISSUE TITLE
Learning rate and batch_size about MiniLM

### ISSUE THREAD

**Issue by LiuChiachi** [microsoft/unilm#360]
Hi @WenhuiWang0824 @donglixp ! I'm training my model using the method MiniLM and I need your help now~
**Describe**
The setup in  MiniLM paper is: 

- learning_rate 6e-4,
- batch_size 512
- using 8 V100 GPUs
- ...

If I use 8 GPUs, what is the learning_rate and batch_size of each card? Should I divide the value above by 8?

Thanks,
jiaqi

**Comment by WenhuiWang0824** [comment]
Hi @LiuChiachi, 

For batch size of each card, please divide by 8 (512/8). The learning rate is 6e-4 for all cards.

Thanks

**Comment by LiuChiachi** [comment]
Thanks a lot:)
